staker-rewards/lib/rewards-calculator.js

```diff
@@ -108,6 +120,21 @@ export default class RewardsCalculator {
     return new BigNumber(minimumStake)
   }
 
+  async isUndelegatingOperator(operatorAddress) {
+    const block = this.interval.startBlock
+
+    const tokenStaking = await this.context.contracts.TokenStaking.deployed()
+
+    const delegationInfo = await callWithRetry(
+      tokenStaking.methods.getDelegationInfo(operatorAddress),
+      block
+    )
+
+    const undelegatedAt = new BigNumber(delegationInfo.undelegatedAt).toNumber()
+
+    return undelegatedAt !== 0 && this.interval.start > undelegatedAt
```

Comment:
Do we need `this.interval.start > undelegatedAt` check?

`getDelegationInfo` is checked for `this.interval.startBlock`. If at that time operator was undelegating, I think it's enough to return `true` from this function.

This check would be needed if we were not calling `getDelegationInfo` at the specific block.

Reply:
Yeah right.